pkgs/development/interpreters/racket/minimal.nix

```
@@ -2,9 +2,9 @@
 }:
 
 racket.overrideAttrs (oldAttrs: rec {
-  name = "racket-minimal-${oldAttrs.version}";
+  pname = "racket-minimal";
+  version = oldAttrs.version;
   src = oldAttrs.src.override {
-    inherit name;
```

Not sure if we want to inherit pname + version here.

---

I am not quite sure what you mean here

---

```suggestion
    inherit pname version;
```

---

sorry but i still do not get it. The snipped does not work since the arguments are not expected.

---

Ahhh, yeah that makes sense.

I am not sure if we want to do then the following or if it does not matter.

```suggestion
    name = "${pname}-${version}";
```